### DISCUSSION TITLE
is my repo featured or getting botted with stars?

### DISCUSSION THREAD

**Discussion by wukko** [community/community#127644] [Repositories] [answered]
### Select Topic Area

Question

### Body

hello! over past few days, https://github.com/imputnet/cobalt got over 2k stars:
<img width="244" alt="image" src="https://github.com/community/community/assets/71202418/baa3eb6b-74ad-42c3-9863-fcae33df2ab7">
<img width="240" alt="image" src="https://github.com/community/community/assets/71202418/69cb8aaf-9188-44b5-8cf6-1849f31e37e4">
*(screenshots from [star history](https://star-history.com/#imputnet/cobalt&Date))*

it spiked my suspicion when i noticed that a bulk of new stars appears every few minutes. i'm wondering whether these stars are real, and in case they're not, is there anything i can do to stop & remove them?

insights only show that a good chunk of visitors came from twitter. but then, where did vast majority (9k unique visitors) come from? why is there no referrer?
<img width="923" alt="image" src="https://github.com/community/community/assets/71202418/7a781380-2ed2-4b27-aa58-11e89035adeb">

really hope i'm worrying about nothing and it's my repo being suddenly popular in china, but just wanna make sure.

**Comment by wukko** [comment, marked as answer]
turns out it’s indeed just trending:
![image](https://github.com/community/community/assets/71202418/c8116fd9-2b33-4c56-96db-7e39579bff9f)

**Comment by ghost** [comment]
Congrats brodie!
